content/metadata.yaml

```diff
@@ -504,4 +504,22 @@ authors:
       string: "None"
       lastapproved: !!str 2020-04-30
     funders: NHGRI R01 HG10067
-
+    
+  -
+    github: bansalvi
+    name: Vikas Bansal
+    initials: VB
+    orcid: 0000-0002-0944-7226
+    twitter: VikasBansal1989
+    email: bansal.bioinfo@gmail.com
+    contributions:
+      - Writing - Original Draft
+      - Investigation
+      - Visualization
```

Comment:
```suggestion
```
We are trying to keep it up to date with what we've already added into the document, so just we just need to remember to add this back in when we add in your visualization!